src/org/opensolaris/opengrok/configuration/Configuration.java

```diff
@@ -1127,7 +1129,7 @@ public String getXMLRepresentationAsString() {
         return bos.toString();
     }
 
-    private void encodeObject(OutputStream out) {
+    public void encodeObject(OutputStream out) {
```

Comment:
I see nothing wrong with exposing this method however am open to suggestions :-)